README.md

```diff
@@ -7,6 +7,17 @@
 This project uses [Yarn](https://yarnpkg.com/getting-started) and [Volta](https://docs.volta.sh/guide/) for package management and version switching.
```

Comment:
@yesyash , I have added document for the same here, Since yarn install on global was a command, I assumed it would be good addition